### ISSUE TITLE
Link to group archives deadends

### ISSUE THREAD

**Issue by mutability** [glidernet/rtlsdr-flarm#8]
The README links to a Google group that is members-only, which isn't too useful.

Could you summarize the discussion referred to, in a generally-accessible location?

**Comment by snip** [comment]
The group is members-only just to get an idea about who is interested by the subject.
Feel free to join us and maybe you can summarize this?
If you need you can follow instructions here http://wiki.glidernet.org/subscribing

**Comment by mutability** [comment]
You're putting a big barrier in the way of anyone interested, but that is your decision. I'll join the group, read the historical discussion, leave the group.

**Comment by mutability** [comment]
(oh, hey, group membership needs to approve it; guess I'm not looking at this this morning!)
